lib/utils/url.js

```diff
@@ -1,4 +1,19 @@
-export const getUtmParams = (query, defaults = {}) =>
+export const getUtmParams = query => {
+  // support /probelesen?campaign=x
+  let defaultCampaign = query.campaign
+  // check matomo attribution cookie
+  if (typeof window !== 'undefined' && window.Matomo) {
```

Comment:
Since you're accessing a property afterwards, aim for a more specific genesis test like `typeof window !== 'object'`?

Reply:
`typeof window === 'object'` would be an alternative but not only objects can have properties (e.g. function can also have them). I prefer the `undefined` check because I do not assume that browser implement it as object. On the other hand they seem to always do that.